.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
```

Comment:
This workflow is to ensure that the code passes the linter and builds successfully. A simple smoke test